PowerEditor/src/Notepad_plus.cpp

```diff
@@ -3947,7 +3947,8 @@ bool Notepad_plus::saveScintillaParams()
 
 bool Notepad_plus::addCurrentMacro()
 {
-	vector<MacroShortcut> & theMacros = (NppParameters::getInstance())->getMacroList();
+	NppParameters * pNppParam = NppParameters::getInstance();
+	vector<MacroShortcut> & theMacros = pNppParam->getMacroList();
 
 	int nbMacro = theMacros.size();
 

```

Comment:
Sorry, I don't want to make to much modification or Don Ho will not accept my PR.